config/subresource-integrity.php

```diff
@@ -1,13 +1,55 @@
 <?php
 
 return [
+    /*
+    |--------------------------------------------------------------------------
+    | Assets Repository Path
+    |--------------------------------------------------------------------------
+    |
+    | This path contains all the asset files used by your application.
+    |
+    */
+
     'base_path' => base_path('public'),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Default Hash Algorithm
+    |--------------------------------------------------------------------------
+    |
+    | This option controls the default algorithm that will be used to hash
+    | assets files used by your application. By default, the sha256 algorithm is
+    | used; however, you remain free to modify this option if you wish.
+    |
+    | Supported: "sha256", "sha384", "sha512"
+    |
+    */
+
     'algorithm' => env('SRI_ALGORITHM', 'sha256'),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Mix SRI Path
+    |--------------------------------------------------------------------------
+    |
+    | This path contains 'mix-sri.json' file generated by Webpack and Laravel
+    | Mix using extensions like 'laravel-mix-sri' witch contains generated
```

Comment:
Minor typo with `which`
```suggestion
    | Mix using extensions like 'laravel-mix-sri' which contains generated
```

Reply:
@pxgamer 
Even with the `which` was not understandable, then I rewrote the message. Now it's better. What do you think, huh?